packages/components/src/components/Address/Address.tsx

```diff
@@ -35,26 +35,25 @@ export const Address: React.FC<AddressProps> = ({
   value,
   copiable = false,
   shortened = false,
-  rpcURL,
+  ens = false,
 }) => {
   const [error, setError] = useState<null | string>(null);
   const [copied, setCopied] = useState<boolean>(false);
   let feedbackTimeOut: ReturnType<typeof setTimeout>;
   let displayAddress: string = value || '';
-  const [ens, setEns] = useState<string | null | undefined>(null);
-  const provider: ethers.providers.JsonRpcProvider | null = rpcURL
-    ? new ethers.providers.StaticJsonRpcProvider(rpcURL)
+  const [ensName, setEnsName] = useState<string | null>(null);
+  const provider: ethers.providers.Web3Provider | null = window.ethereum
+    ? new ethers.providers.Web3Provider(window.ethereum, 'any')
```

Comment:
This will only work if there's an active wallet connection though. That's not ideal imo

Reply:
Made the changes to accept the provider as a prop and added to resolve an error i am getting while trying to use the component